cmd/tail/tail.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"teleport-exec/file_stream"
+	"time"
+)
+
+//-------------------------------------------------------------------------------------------------
+func main() {
+	if len(os.Args) < 2 {
+		log.Fatalln("Need an argument!")
+	}
+	file_name := os.Args[1]
+	log.Println("Tailing file:", file_name)
+
+	// Timeout tailing after a while
+	timeout_ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
I get that this is a toy program, but don't forget to support cancelation (probably by handling SIGINT)